src/w3c/templates/headers.js

```diff
@@ -274,6 +275,58 @@ export default (conf, options) => {
   </div>`;
 };
 
+function renderFeedback(conf) {
+  if (!conf.github && !conf.wgPublicList) return;
+  const definitions = [];
+
+  // Github feedback...
+  if (conf.github) {
+    const { repoURL, issuesURL, newIssuesURL, pullsURL, fullName } =
+      conf.github;
+    definitions.push(
+      html`<dd>
+        <a href="${repoURL}">GitHub ${fullName}</a>
+        (<a href="${pullsURL}">pull requests</a>,
+        <a href="${newIssuesURL}">new issue</a>,
+        <a href="${issuesURL}">open issues</a>)
+      </dd>`
+    );
+  }
+
+  if (conf.wgPublicList) {
+    // The <a href="mailto:list?subject"> link for the public list
+    const mailToURL = new URL(`mailto:${conf.wgPublicList}@w3.org`);
+    const subject =
+      conf.subjectPrefix ?? `[${conf.shortName}] ${l10n.your_topic_here}`;
+    const mailingListLink = html`<a
+      href="${mailToURL.href}?subject=${encodeURIComponent(subject)}"
+      >${mailToURL.pathname}</a
```

Comment:
Should we remove some unnecessary parts from the pathname? Like `/Archives/Public/`?

Reply:
Not following? ... let me check.

Reply:
This is how it renders with with a subject:  

![Screen Shot 2021-10-01 at 11 16 37 am](https://user-images.githubusercontent.com/870154/135550954-9a8e9533-d810-40ca-9960-0dc0e3855c1f.png)

And without: 
![Screen Shot 2021-10-01 at 11 17 09 am](https://user-images.githubusercontent.com/870154/135550978-a6b63837-2379-482a-a85d-c219d80d83c8.png)
 
I think it's fine as it is... still a bit confused about `/Archives/Public/`?